netrd/reconstruction/marchenko_pastur.py

```diff
@@ -148,9 +142,9 @@ def fit(
         selected = (w < w_min) | (w > w_max)
 
         if selected.sum() == 0:
-            G = nx.empty_graph(n=N)
-            self.results['graph'] = G
-            return G
+            self.update_matrix(np.zeros((N, N)))
```

Comment:
Can we use sparse matrices for dummy matrices like this? Or is it too clunky later?